File: ItemCount/Module/CS/ItemCount.Module/BusinessObjects/Email.cs
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Xpo;

namespace ItemCount.Module.BusinessObjects {
    [DefaultClassOptions]
    public class Email : BaseObject {
        public Email(Session session) : base(session) { }

        string name;
        public string Name {
            get { return name; }
            set { SetPropertyValue(nameof(Name), ref name, value); }
        }
    }
}

File: ItemCount/Module/VB/ItemCount.Module/BusinessObjects/Email.vb
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Xpo

<DefaultClassOptions>
Public Class Email
    Inherits BaseObject

    Public Sub New(ByVal session As Session)
        MyBase.New(session)
    End Sub

    Private _Name As String

    Public Property Name As String
        Get
            Return _Name
        End Get
        Set(ByVal value As String)
            SetPropertyValue(NameOf(Name), _Name, value)
        End Set
    End Property
End Class
